includes/class-main.php

```
@@ -498,12 +498,17 @@ public function admin_enqueue_scripts(): void {
 		}
 
 		$asset_file = WPPO_PLUGIN_PATH . 'build/index.asset.php';
+		$resolved   = wp_normalize_path( realpath( $asset_file ) );
 
-		// Include the asset file to retrieve dependencies and version.
-		$asset_data = file_exists( $asset_file ) ? include $asset_file : array(
-			'dependencies' => array(),
-			'version'      => false,
-		);
+		// Validate the resolved path is within the plugin directory before including.
+		if ( false !== $resolved && strpos( $resolved, WPPO_PLUGIN_PATH ) === 0 ) {
```

PHPStan (2.2.5)

[warning] Constant WPPO_PLUGIN_PATH not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)